product_standard_margin/models/product_product.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (c) 2012 Camptocamp SA (http://www.camptocamp.com)
+#    All Right Reserved
+#
+#    Author : Joel Grand-Guillaume
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import fields, api
+from openerp.models import Model
+import openerp.addons.decimal_precision as dp
+
+
+class ProductProduct(Model):
+    _inherit = 'product.product'
+
+    @api.multi
+    @api.depends(
+        'product_tmpl_id.list_price', 'replenishment_cost', 'taxes_id.type',
+        'taxes_id.price_include', 'taxes_id.amount',
+        'taxes_id.include_base_amount', 'taxes_id.child_depend')
+    def _get_margin(self):
+        tax_obj = self.pool['account.tax']
+        for product in self:
+            product.list_price_vat_excl = tax_obj.compute_all(
```

Comment:
There is a `@api.v8` version of this method https://github.com/odoo/odoo/blob/dd4566de37559697af68d2239429e460c16e17d1/addons/account/account.py#L2140-L2144